recipe/meta.yaml

```diff
@@ -25,35 +25,37 @@ build:
 requirements:
   host:
     - python
-    - debugpy >=1.0.0
-    - hatchling >=1.4
-    - ipython >=7.23.1
-    - jupyter_client >=6.1.12
-    - packaging
     - pip
-    - comm >=0.1
-    - psutil
+    - setuptools
     - wheel
-  run:
-    - appnope  # [osx]
+    - debugpy >=1.6.5
     - comm >=0.1.1
-    - debugpy >=1.0
+    - hatchling
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
-    - matplotlib-inline >=0.1
-    - nest-asyncio
+    - jupyter_core >=4.12,!=5.0.*
     - packaging
     - psutil
```

Comment:
Do we really need all these packages in `host` except `hatchling` and `jupiter_client` for building the package, see https://github.com/ipython/ipykernel/blob/bed7ee4721c8295d61940aae60f2b78d172d2f33/pyproject.toml#L2

Reply:
I have to also include `comm`, `ipython`, and `psutil`. Without them I get a `ModuleNotFoundError`